CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## XXX
```

Comment:
Please @arneee don't update the changelog because it is updated automatically when the code is released. Thanks for your support!